contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol

```diff
@@ -72,9 +78,11 @@ struct LimitsListPacked {
     uint16 maxValidatorExitRequestsPerReport;
     uint64 requestTimestampMargin;
     uint64 maxPositiveTokenRebase;
+    uint64 maxExitedValidatorsRatePerDay;
```

Comment:
maybe extra because we already have a churn limit (see above)

Reply:
it's checked as a part of `Lido.handleOracleReport`, right?

Reply:
nope, I only appearing validators

Reply:
then I'd keep it